design/airgap-existing-cluster.md

```diff
@@ -0,0 +1,109 @@
+# Support for installing KOTS Admin Console in Airgapped Clusters
+
+## Goals
+
+- Install kotsadm to an existing cluster where the cluster does not have internet access
+- Install kotsadm to an existing cluster from a workstation where the workstation does not have internet access at the time of installation
+- Support different credentials for push / pull (workstation / cluster) to the local registry
+- Support temporary/dynamic and persistent/static credentials for pushing kotsadm images to the registry
+- Support only persistent/static credentials for pulling kotsadm images to the registry
+- Support different credentials for pushing vs pulling images to the local registry
+
+## Non Goals
+
+- Create a single installer for kotsadm + application
+- Create a single download for kotsadm, kots plugin, application and license
+- Automated installs of kotsadm to existing clusters in airgap mode
+- Support for installing versions of KOTS different from the KOTS CLI version installed
+
+## Background
+
+It's not currently possible to install KOTS Admin Console into an existing cluster that doesn't have outbound internet access.
+
+## High-Level Design
+
+Airgap Workstation: the computer with kubectl access to the airgap cluster
+Public Workstation: the computer with access to the internet
+
+From the public workstation:
+1. The user downloads the kots plugin suitable for Airgap Workstation.
+1. The user downloads the kotsadm archive containing kots container images
+1. The user downloads the airgap bundle of the application
+1. The user copies the kots plugin, the kotsadm archive and the application airgap bundle onto a medium (eg a USB stick) that can be transfered into the airgapped network.
+
+From the airgap workstation:
+1. The user will install the kots plugin 
```

Comment:
Is this a manual `cp` command or do we provide a script to do this?

Reply:
To capture: We talked and agreed to have an install script like kurl airgapped install.